cdap-api/src/main/java/co/cask/cdap/api/schedule/Schedule.java

```diff
@@ -67,33 +67,29 @@ public String getCronEntry() {
 
   @Override
   public boolean equals(Object o) {
-    if (this == o) {
-      return true;
-    }
-    if (o == null || getClass() != o.getClass()) {
-      return false;
-    }
+    if (this == o) return true;
+    if (o == null || getClass() != o.getClass()) return false;
 
     Schedule schedule = (Schedule) o;
 
-    if (cronEntry.equals(schedule.cronEntry) && description.equals(schedule.description)
-      && name.equals(schedule.name)) {
-      return true;
-    }
-    return false;
+    if (cronEntry != null ? !cronEntry.equals(schedule.cronEntry) : schedule.cronEntry != null) return false;
+    if (description != null ? !description.equals(schedule.description) : schedule.description != null) return false;
+    if (name != null ? !name.equals(schedule.name) : schedule.name != null) return false;
```

Comment:
Isn't that would fail checkstyle? Can you just revert the change by checking out the old commit for the file?